assets/src/edit-story/components/canvas/editElement.js

```diff
@@ -44,17 +47,47 @@ function EditElement({ element }) {
     getBox: state.actions.getBox,
   }));
 
-  const { Edit } = getDefinitionForType(type);
+  const [editWrapper, setEditWrapper] = useState(null);
+  const [actionHappening, setActionHappening] = useState(false);
+
+  const { lastSelectionEvent } = useCanvas(
```

Comment:
`lastSelectionEvent`, I believe, is used only explicitly for dragging. Since we're disabling it, we should drop it from here as well.